src/vs/base/common/event.ts

```diff
@@ -947,10 +947,11 @@ export class Emitter<T> {
 					this._listeners = new LinkedList();
 				}
 
-				if (this._leakageMon && this._listeners.size > this._leakageMon.threshold * 3) {
-					console.warn(`[${this._leakageMon.name}] REFUSES to accept new listeners because it exceeded its threshold by far`);
-					return Disposable.None;
-				}
+				// {{SQL CARBON EDIT}} Disabling threshold to unblock model view components to register more listeners. Due to the threshold, the components were not able to register listeners and were not able to handle click events.
+				// if (this._leakageMon && this._listeners.size > this._leakageMon.threshold * 3) {
```

Comment:
What about adding a telemetry event here (with the callstack) so we can track when this is happening? It'd be good to know where else this might be happening that we just aren't noticing or haven't run in to yet.

Reply:
Since it is in base, adding telemetry will require some bigger changes since I cannot just use the service. I will just merge this in for now and take those as a follow up item.

Reply:
Ok, unfortunately I looked at it and I don't see a straightforward way to inject the telemetry service into these emitters. And we can't even do something like pop up a toast for the same reason....

Let me think about it and I'll let you know if I come up with any ideas.